api/models/constants.py

```diff
@@ -1,2 +1,43 @@
 DEFAULT_CHAR_FIELD_MAX_LEN = 255
 DEFAULT_CHOICE_CHAR_FIELD_MAX_LENGTH = 15
+
+# IMPORTANT: the boundaries of larger ranges must match the boundaries of the smaller ones
+
+# Match string ranges that are presented to the user to actual ranges, e.g.:
+# {"6-8": (6, 9), "9-11": (9, 12), ...}
+STUDENT_AGE_RANGE_BOUNDARIES = {
```

Comment:
I advise to rename it to `STUDENT_AGE_RANGES`. It will have same meaning, but shorter  => easier to read

Same below for teachers'

Reply:
OK. My idea was that `...RANGES` would imply `range()` objects, which could be confusing if there are no `range` objects there